SPECS/patchelf/patchelf.spec

```diff
@@ -0,0 +1,178 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+# hardening breaks the set-interpreter-long test on i686, x86_64, ppc64le, s390x
+%undefine _hardened_build
```

Comment:
Please build without disabling hardened builds. We have enabled these everywhere and don't want to make exceptions as far as I know. Especially since the comment seems to only mention test issues. I believe Henry Beberman might have more details on the topic, if needed.